qemu/tests/qemu_guest_agent.py

```diff
@@ -2553,7 +2553,7 @@ def gagent_check_fsinfo(self, test, params, env):
             error_context.context("Check file system type of '%s' mount point." %
                                   mount_pt, logging.info)
             fs_type_qga = fs["type"]
-            cmd_get_disk = params["cmd_get_disk"] % mount_pt
+            cmd_get_disk = params["cmd_get_disk"] % mount_pt.replace("/", r"\/")
```

Comment:
Hello @xiagao, I updated the pattern of "cmd_get_disk" to make it can handle multiple "/" like "/boot/efi". Can you run "qemu_guest_agent.virtio_serial.check_fsinfo" to ensure this change will not block your test?

Reply:
> Hello @xiagao, I updated the pattern of "cmd_get_disk" to make it can handle multiple "/" like "/boot/efi". Can you run "qemu_guest_agent.virtio_serial.check_fsinfo" to ensure this change will not block your test?

@6-dehan Could you help to check it?

Reply:
Hello @6-dehan, any update?

Reply:
hello @PaulYuuu ,I've run this case and the result is  
(1/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsinfo.q35: ERROR: Shell command failed: "cat /proc/mounts |grep -v rootfs |awk '$2~/^\\\\/boot$/{print $1,$3}'"    (status: 1,    output: 'awk: cmd. line:1: $2~/^\\\\/boot$/{print $1,$3}\nawk: cmd. line:1:               ^ unterminated regexp\n') (79.35 s)
I checked qemu_guest_agent.cfg and found that the cmd is :cat /proc/mounts |grep -v rootfs |awk '$2~/^\%s$/{print $1,$3}',do you notice that its already has escape character '\' so that you seem don't need to replace anything.don't you ? well,this is my opinion.

Reply:
As I mentioned, the previous code can only handle single `/`. If a guest was installed through UEFI, there has a path "/boot/efi", then this case will fail.

> ERROR: Shell command failed: "cat /proc/mounts |grep -v rootfs |awk '$2~/^\\/boot$/{print $1,$3}'" (status: 1, output: 'awk: cmd. line:1: $2~/^\\/boot$/{print $1,$3}\nawk: cmd. line:1: ^ unterminated regexp\n')

For this ERROR, maybe you forget to modify `cmd_get_disk = cat /proc/mounts |grep -v rootfs |awk '$2~/^%s$/{print $1,$3}'` in cfg.

Reply:
right,I thought maybe I should deleted '\\' from 'cmd_get_disk = `cat /proc/mounts |grep -v rootfs |awk '$2~/^%s$/{print $1,$3}`' ,then your updating makes sense,the result was passed.

Reply:
Well, you can use `curl -L https://github.com/autotest/tp-qemu/pull/2404.patch | git am` to apply the patch, and then you won’t miss some related changes.

Reply:
(1/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsinfo.q35: PASS (78.81 s)

Reply:
Thank you for the update.